scio-bigquery/src/main/scala/com/spotify/scio/bigquery/types/ConverterProvider.scala

```diff
@@ -153,7 +153,7 @@ private[types] object ConverterProvider {
 
     val tn = TermName("r")
     q"""(r: _root_.org.apache.avro.generic.GenericRecord) => {
-          import _root_.scala.jdk.CollectionConverters._
```

Comment:
Wait we were using 2.13 converters this whole time? That should've broken 2.12 tests?

Reply:
Never mind. Looks like these are from `scala-collection-compat`. We should fix it for `scio-avro` macros too right?

Reply:
the tests were also using `_root_.scala.jdk.CollectionConverters._` so no problem there ... and by luck most of ppl where not importing `_root_.scala.jdk.CollectionConverters._`